<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ripes::MemoryViewerWidget</class>
 <widget class="QWidget" name="Ripes::MemoryViewerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="MemoryView" name="memoryView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="RadixSelectorWidget" name="radixSelector" native="true"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Go to:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="GoToComboBox" name="gotoCombobox"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RadixSelectorWidget</class>
   <extends>QWidget</extends>
   <header>radixselectorwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MemoryView</class>
   <extends>QTableView</extends>
   <header>memoryview.h</header>
  </customwidget>
  <customwidget>
   <class>GoToComboBox</class>
   <extends>QComboBox</extends>
   <header>gotocombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
